### DISCUSSION TITLE
Userpass authentication method named differently

### DISCUSSION THREAD

**Discussion by davetrux** [quarkiverse/quarkus-vault#173] [Q&A] [answered]
In our instance of Vault, the admin named the userpass authentication method something different than "userpass".

Using curl, I can login by changing the path from `v1/auth/userpass/login` to `/v1/auth/othername/login/` and it works.

I have not been able to find a configuration for our Quarkus app that works. The app cannot connect, it just gets 403 responses back from Vault, because it's trying to use `v1/auth/userpass/login`.

In our Quarkus app, what can I configure to have the same effect as the curl login?

**Comment by kdubb** [comment, marked as answer]
@davetrux This has been implemented and is available in 4.0.1 with the parameter `quarkus.vault.authentication.userpass.authMountPath`
